src/v/cluster/controller.h

```diff
@@ -34,6 +33,10 @@
 #include <chrono>
 #include <vector>
 
+namespace cloud_storage {
+class topic_mount_handler;
+}
```

Comment:
shouldn't it go to `src/v/cloud_storage/fwd.h` and this file to be included here?

Reply:
I mean, why this change?

Reply:
I prefer forward declaration when I need just a single class or two to including a separate file with unrelated things. I don't want changing that single file causing recompilations for unrelated things.